internal/server/iigointernal/executive.go

```diff
@@ -0,0 +1,148 @@
+package iigointernal
+
+import (
+	"github.com/SOMAS2020/SOMAS2020/internal/common/baseclient"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/gamestate"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/roles"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/voting"
+)
+
+type executive struct {
```

Comment:
what's `executive`?

Reply:
The head of the legislative branch is the Speaker, the head of the executive branch is the President, and the head of the judicial branch is the Judge